### ISSUE TITLE
Log button isn't filtering by Peer (Resource Group)

### ISSUE THREAD

**Issue by Hou5e** [FoldingAtHome/fah-web-client-bastet#59]
Linux Mint 21.1 (U22.04), FAH Client v8.1.5:
When you press the log button for a Peer (Resource Group) where the Peer subtitle updates but the search bar isn't filled-in for the filter text to make it work.

The 'Log' button probably needs to be like the 'WU Log' button below it, and pass the search filter text at the end of the URL, like: ?q=:/RG2:   so it would be like this:  https://app.foldingathome.org/2/log?q=:/RG2:

![image](https://user-images.githubusercontent.com/20870053/209449981-e857c72d-7ccd-427a-9571-552f50b2be23.png)
![image](https://user-images.githubusercontent.com/20870053/209449990-e62e1486-1e53-414c-b7b1-cbbdade2b209.png)

**Comment by kbernhagen** [comment]
Note that not all log lines for a group contain `:/group:`.
Some lines do not even have `/group` without colons. E.g.
```
08:03:57:D3:/gpu1:New client from 127.0.0.1:56525
08:03:57:D3:'/gpu1' msg: {
08:03:57:D3:  "cmd": "config",
08:03:57:D3:  "config": {"priority": "normal"}
08:03:57:D3:}
08:03:57:D3:/gpu1:Closing client from 127.0.0.1:56525
```

**Comment by jcoffland** [comment]
Should be fixed in v8.3.x
